lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -559,6 +567,12 @@ abstract class AssetPickerBuilderDelegate<Asset, Path> {
   /// Yes, the build method.
   /// 没错，是它是它就是它，我们亲爱的 build 方法~
   Widget build(BuildContext context) {
+    if (_savedScrollPosition != null) {
```

Comment:
Maybe we should use `ScrollController(initialScrollOffset: yourSavedScrollPosition)` instead.